### ISSUE TITLE
Minecraft has crashed! (Java Runtime Problems)

### ISSUE THREAD

**Issue by Teeenoob** [simply-kel/AlinLib#8]
Crash .log
**Playing on 1.20.4**
```javascript
Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.UnsupportedClassVersionError: net/minecraft/class_310 has been compiled by a more recent version of the Java Runtime (class file version 65.0), this version of the Java Runtime only recognizes class file versions up to 61.0
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:63) ~[fabric-loader-0.15.11.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:472) ~[fabric-loader-0.15.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.11.jar:?]
Caused by: java.lang.UnsupportedClassVersionError: net/minecraft/class_310 has been compiled by a more recent version of the Java Runtime (class file version 65.0), this version of the Java Runtime only recognizes class file versions up to 61.0
	at net.minecraft.client.main.Main.main(Main.java:233) ~[1.20.4-0.15.11.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.11.jar:?]
	... 2 more
```

**Comment by kel-cu** [comment]
I recommend upgrading to version 21 of Java

**Comment by kel-cu** [comment]
Fixed with 1.2.0-alpha.6.1
